recipes/libwayland/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class WaylandConan(ConanFile):
+    name = "libwayland"
+    description = "Wayland is a project to define a protocol for a compositor to talk to its clients as well as a library implementation of the protocol"
+    topics = ("conan", "wayland")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://wayland.freedesktop.org"
+    license = "MIT"
+    generators = "pkg_config"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "enable_libraries": [True, False],
```

Comment:
```suggestion
        "enable_libffi": [True, False],
```
I think more explicit sounds better

Reply:
this is actually about enabling [wayland libraries,](https://github.com/wayland-project/wayland/blob/1.18.0/meson_options.txt#L2) the fact that it requires libffi is an implementation detail

Reply:
So the package should be called wayland instead.